slashing/program/program-id.md

```diff
@@ -0,0 +1 @@
+8sT74BE7sanh4iT84EyVUL8b77cVruLHXGjvTyJ4GwCe
```

Comment:
Just used the default key when from when I deployed to testnet, can update once we decide how this is going to be deployed.

Reply:
Got it, let me know if I have this correct for the deployment scheme:
- We upload the program to some address A
- On epoch boundary where the feature flag is active we create a program account owned by the system program `S1ashing1111...`
- We them move the contents of A to this vanity address
- Further upgrades to the program will require another feature flag

I did see that some of the other spl programs were deployed with a squads multi sig as the upgrade authority. Curious about your thoughts about that deployment schema. 

It seems that would allow us more freedom in upgrading or disabling the slashing program should there be any vulnerabilities. But optically is it worse if the keys are held by anza & FD?

Reply:
Yeah, that would be the process. Anza / FD really can't hold the upgrade keys to something so sensitive -- we'll need to leave it up to the validators. We can start off with feature gates, and then eventually move to a validator DAO, if that ever materializes.